### ISSUE TITLE
Are 360 annotations used for CAMERA_ALL evaluation in test set?

### ISSUE THREAD

**Issue by nhatnxm** [waymo-research/waymo-open-dataset#329]
If modality is CAMERA ONLY, model cann't perceive object of 360 view, especially the back behind the car. I wonder if 360 annotations are used to evaluate model trained by CAMERA ONLY or, do you only use annotations that are in the field of view of cameras?

**Comment by peisun1115** [comment]
yes, 360 annotations are used. those annotations that are not visible in cameras are considered FNs which is true for all CAMERA_ALL models. So the metrics should still be valid.
